### PR TITLE
Add PID to table, add assessed building(s) to property slideout

### PR DESCRIPTION
**Pull request by jdtoombs**
Add PID to inventory list view, and also disable animation on another zoom button as well
Also add assessed building(s) to property slideout

### CODE REVIEW

File: frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx [hunk 1]
```diff
@@ -31,6 +31,15 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo, canV
     formatAssessed = '';
   }
 
+  let improvments = 0;
```
Comment (jdtoombs) [inline frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx:34]: I just copied what was done above for assessed land. It appears that for all cases of the property slideout it is missing `assessedLand` and `assessedBuilding` as it takes in an `IParcel` or `IBuilding` as a prop not `IProperty`

File: frontend/src/components/maps/leaflet/InfoSlideOut/ThreeColumnItem.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const InnerRow = styled(Row)`
 
 const LeftCol = styled(Col)`
   width: 110px;
-  max-width: 110px;
+  max-width: 135px;
```
Comment (jdtoombs) [inline frontend/src/components/maps/leaflet/InfoSlideOut/ThreeColumnItem.tsx:13]: Just so it would fit Assessed Building(s) without wrapping

**Review by Fosol — APPROVED** [review]

**Comment by mbertucci** [comment]
You guys are killin' it!!!

File: frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx [hunk 3]
```diff
@@ -31,6 +31,17 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo, canV
...
```
Comment (devinleighsmith) [inline frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx:34]: /nit improvements

File: frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx [hunk 4]
```diff
@@ -31,6 +31,17 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo, canV
...
+  if (parcelInfo?.assessedBuilding) {
+    improvments = parcelInfo?.assessedBuilding;
+  } else if (parcelInfo?.buildings?.length >= 1) {
+    parcelInfo.buildings.forEach((building: IBuilding) => {
+      improvments += +building.evaluations
+        .sort((a: IEvaluation, b: IEvaluation) => compareDate(a.date, b.date))
+        .reverse()[0].value;
```
Comment (devinleighsmith) [inline frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx:41]: this looks unsafe to me, what happens if there are no evaluations?

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx (hunk 4 above)
Comment (devinleighsmith) [reply frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx:41]: @jdtoombs

Comment (jdtoombs) [reply frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx:41]: I'll change the check to confirm there are evaluations!

**Review by goemen — APPROVED** [review]

**Pull request merged**